quick-start.md

```diff
@@ -13,7 +13,7 @@ has_toc: false
 
 ### Overview
 
-Open Horizon is a middleware solution designed to manage the deployment and service software lifecycle of containerized applications. It also supports an optional separation of concerns between a service and related machine learning assets by providing a separate [model sync service](developing/model_management_details.md#model_management_details).  These solutions enable autonomous management of large fleets of edge compute nodes in various connected states.
+Open Horizon is a middleware solution designed to manage the deployment and service software lifecycle of containerized applications. It also supports an optional separation of concerns between a service and related machine learning assets by providing a separate [model sync service](/docs/developing/model_management_details.md#model_management_details).  These solutions enable autonomous management of large fleets of edge compute nodes in various connected states.
```

Comment:
I'm not certain if the `.md` should be included in the URL.  Let's see if page processing fixes it up.